### DISCUSSION TITLE
how to send report to slack ?

### DISCUSSION THREAD

**Discussion by drorvt** [zricethezav/gitleaks#681] [General]
how to send report to slack ?

**Comment by w0rmr1d3r** [comment]
Hello @drorvt !

I believe currently gitleaks doesn't offer that functionality.
However, there might be someone who has done that within their systems.

Where are you running gitleaks? There might be already an integration for that.

Cheers!

**Comment by drorvt** [comment]
In Gotland

**Comment by w0rmr1d3r** [comment]
As seen in this announcement -> https://github.com/zricethezav/gitleaks/issues/683
gitleaks won't be responsible for sending reports to Slack.
